website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -0,0 +1,75 @@
+# `invoke`
+
+```python
+def invoke(
+    contract_address: int,
+    function_name: str,
+    inputs: list[int] | dict[str, Any] | None = None,
+    max_fee: Optional[int] = None, # in Wei
+    auto_estimate_fee: bool = False,
```

Comment:
Other migrator cheatcodes accept config dictionary as a last argument. This cheatcode breaks this "convention".

Reply:
This has been fixed